.teamcity/jdks.yaml

```diff
@@ -1,145 +1,161 @@
 version: v1
 jdks:
-  - param: "linux.java7.oracle.64bit"
+  - param: linux.java7.oracle.64bit
     os: linux
     arch: amd64
-    vendor: "oracle"
-    version: "7u80"
+    vendor: oracle
+    version: 7u80
     sha256: bad9a731639655118740bee119139c1ed019737ec802a630dd7ad7aab4309623
-  - params: [ "linux.java8.openjdk.64bit", "linux.java8.oracle.64bit" ]
+  - params:
+      - linux.java8.openjdk.64bit
+      - linux.java8.oracle.64bit
     os: linux
     arch: amd64
-    vendor: "oracle"
-    version: "8u401"
+    vendor: oracle
+    version: 8u401
     sha256: 19684fccd7ff32a8400e952a643f0049449a772ef63b8037d5b917cbd137d173
-  - param: "linux.java11.openjdk.64bit"
+  - param: linux.java11.openjdk.64bit
     os: linux
     arch: amd64
-    vendor: "temurin"
-    version: "jdk-11.0.23+9"
-    sha256: 23e47ea7a3015be3240f21185fd902adebdcf76530757c9b482c7eb5bd3417c2
-  - params: [ "env.JAVA_HOME", "linux.java17.openjdk.64bit" ]
+    vendor: temurin
+    version: jdk-11.0.25+9
```

Comment:
:x: Please use quotes for string fields to avoid potential YAML shenanigans (e.g. interpreting strings as numbers)